recipes/esl.rb

```diff
@@ -21,6 +21,8 @@
 
 case node['platform_family']
 when 'debian'
+  package 'apt-transport-https'
```

Comment:
Is this to work around the docker images not having this? It *should* be on every distro install in the real world